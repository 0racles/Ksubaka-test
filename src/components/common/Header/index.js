import React from 'react';
import styled from 'styled-components';
import logo from './logo.png';
import Config from '../../../config';

const Wrapper = styled.div`
  background-color: ${Config.theme.colors.pink};
  color: ${Config.theme.colors.white};
  height: 9.375rem;
  padding: 1.25rem;
`;

const StyledImage = styled.img`
  height: 5rem;
`;

const Header = () => {
  return (
    <Wrapper className="header-title">
      <StyledImage src={logo} className="header-logo" alt="logo" />
      <h2>Ksubaka Tech Assessment</h2>
    </Wrapper>
  );
};

export default Header;
