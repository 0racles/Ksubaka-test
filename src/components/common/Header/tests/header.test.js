import Header from '../index';

describe('Header', () => {
  it('should match snapshot', () => {
    const renderedItem = shallow(<Header />);
    expect(renderedItem).toMatchSnapshot();
  });
});
