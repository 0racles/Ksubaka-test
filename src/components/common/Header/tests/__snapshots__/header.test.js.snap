// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<.sc-bdVaJa
  className="header-title"
>
  <.sc-bwzfXH
    alt="logo"
    className="header-logo"
    src="logo.png"
  />
  <h2>
    Ksubaka Tech Assessment
  </h2>
</.sc-bdVaJa>
`;
